python/cli/prelude_cli/views/iam.py

```diff
@@ -25,6 +25,18 @@ def register_account(controller):
     print_json(data=creds)
 
 
+@iam.command('update-account')
+@click.option('-m', '--mode',
+              help='provide a mode',
+              default='manual', show_default=True,
+              type=click.Choice(['manual', 'frozen'], case_sensitive=False))
+@click.pass_obj
+@handle_api_error
+def update_account(controller, mode):
+    """ Update an account """
+    controller.update_account(mode=mode)
```

Comment:
sdk param is listed as an int, what's this?

Reply:
That's me flipping between a string, an enum and an int, followed by you having a better eye than me!